set.mm

```diff
@@ -522922,6 +522938,70 @@ Real and complex numbers (cont.)
+     by Scott Fenton, 7-Aug-2024.) $)
+  rexbi $p |- ( A. x e. A ( ph <-> ps ) ->
+  ( E. x e. A ph <-> E. x e. A ps ) ) $=
+    ( wb wral cv wcel wa wex wrex wal wi df-ral pm5.32 biimpi alimi exbi df-rex
+    sylbi syl 3bitr4g ) ABEZCDFZCGDHZAIZCJZUEBIZCJZACDKBCDKUDUFUHEZCLZUGUIEUDUE
+    UCMZCLUKUCCDNULUJCULUJUEABOPQTUFUHCRUAACDSBCDSUB $.
+
+  ${
+    $d F x $.  $d A x $.  $d B x $.
+    $( The image of a function under a singleton.  (Contributed by Scott
+       Fenton, 8-Aug-2024.) $)
+    fvimasn $p |- ( ( F Fn A /\ B e. A ) -> ( F " { B } ) = { ( F ` B ) } ) $=
+      ( vx wfn wcel wa csn cima cv wbr cab wceq imasng adantl velsn eqcom bitri
+      cfv fnbrfvb syl5rbb abbi1dv eqtrd ) CAEZBAFZGZCBHIZBDJZCKZDLZBCSZHZUEUGUJ
+      MUDDBACNOUFUIDULUHULFZUKUHMZUFUIUMUHUKMUNDUKPUHUKQRABUHCTUAUBUC $.
+  $}
+
+  ${
+    $d A y $.  $d ph y $.  $d x y $.
+    riotarab.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Restricted iota of a restricted abstraction.  (Contributed by Scott
+       Fenton, 8-Aug-2024.) $)
+    riotarab $p |- ( iota_ x e. { y e. A | ps } ch ) =
+     ( iota_ x e. A ( ph /\ ch ) ) $=
+      ( cv crab wcel wa cio crio wb weq bicomd equcoms elrab anbi1i df-riota
+      anass bitri iotabii 3eqtr4i ) DHZBEFIZJZCKZDLUEFJZACKZKZDLCDUFMUJDFMUHUKD
+      UHUIAKZCKUKUGULCBAEUEFBANDEDEOABGPQRSUIACUAUBUCCDUFTUJDFTUD $.
+  $}
+
+  ${
+    $d A y $.  $d ph y $.  $d x y $.
+    reurab.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Restricted existential uniqueness of a restricted abstraction.
+       (Contributed by Scott Fenton, 8-Aug-2024.) $)
+    reurab $p |- ( E! x e. { y e. A | ps } ch <-> E! x e. A ( ph /\ ch ) ) $=
+      ( cv crab wcel wa weu wreu wb weq bicomd equcoms elrab anbi1i df-reu
+      anass bitri eubii 3bitr4i ) DHZBEFIZJZCKZDLUEFJZACKZKZDLCDUFMUJDFMUHUKDUH
+      UIAKZCKUKUGULCBAEUEFBANDEDEOABGPQRSUIACUAUBUCCDUFTUJDFTUD $.
+  $}
+
+  ${
+    snres0.1 $e |- B e. _V $.
+    $( Condition for restriction of a singleton to be empty.  (Contributed by
+       Scott Fenton, 9-Aug-2024.) $)
+    snres0 $p |- ( ( { <. A , B >. } |` C ) = (/) <-> -. A e. C ) $=
+      ( cop csn cres c0 wceq cdm cin wcel wn wrel wb relres reldm0 ax-mp dmsnop
+      dmres ineq2i eqtri eqeq1i disjsn 3bitri ) ABEFZCGZHIZUGJZHIZCAFZKZHIACLMU
+      GNUHUJOUFCPUGQRUIULHUICUFJZKULUFCTUMUKCABDSUAUBUCCAUDUE $.
+  $}
+
+  $( Subset relationship for the domain of a restriction.  (Contributed by
+     Scott Fenton, 9-Aug-2024.) $)
+  resdmss $p |- dom ( A |` B ) C_ B $=
+    ( cres cdm cin dmres inss1 eqsstri ) ABCDBADZEBABFBIGH $.
+
+
+  $( The union of two ordinals is an ordinal.  (Contributed by Scott Fenton,
+     9-Aug-2024.) $)
+  onun2 $p |- ( ( A e. On /\ B e. On ) -> ( A u. B ) e. On ) $=
+    ( con0 wcel wa wss wceq ssequn1 wi eleq1a adantl syl5bi ssequn2 adantr word
+    cun wo eloni ordtri2or2 syl2an mpjaod ) ACDZBCDZEZABFZABPZCDZBAFZUEUFBGZUDU
+    GABHUCUIUGIUBBCUFJKLUHUFAGZUDUGBAMUBUJUGIUCACUFJNLUBAOBOUEUHQUCARBRABSTUA
+    $.
```

Comment:
Similarly, [`~onun2i`](https://us.metamath.org/mpeuni/onun2i.html) can be proved using this one.

Reply:
Done